examples/variational_autoencoder.py

```diff
@@ -40,8 +37,8 @@ def sampling(args):
 x_decoded_mean = decoder_mean(h_decoded)
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
-    kl_loss = - 0.5 * K.mean(1 + z_log_std - K.square(z_mean) - K.exp(z_log_std), axis=-1)
+    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
The reconstruction term in the variational objective function is `E[- ln p(x|z)]`. Since we assume element-wise independence, this becomes `E[\sum -\ln p(x_i | z)]`. Thus, we need to sum the cross-entropy terms over all 784 dimensions. The implementation of `objectives.binary_crossentropy` averages over all dimensions (except the batch dimension). Thus, I multiplied.